src/components/ResourceList/components/BulkActions/BulkActions.tsx

```diff
@@ -169,6 +169,7 @@ class BulkActions extends React.PureComponent<CombinedProps, State> {
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
ESLint wants us to put private methods below public ones like render(). However in this file that'll make the diff incredibly noisy so I've surpassed the warning